JetNews/app/src/main/java/com/example/jetnews/ui/PreviewUtils.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.jetnews.ui
```

Comment:
Copyright

Reply:
Should it be attributed to Google, Inc even if I created the file?

Reply:
Yes please.

Reply:
I'll go with that for now...